lib/email/receiver.rb

```diff
@@ -1100,21 +1102,28 @@ def notify_about_rejected_attachment(attachments)
       Email::Sender.new(client_message, :email_reject_attachment).send
     end
 
-    def create_post(options = {})
-      options[:via_email] = true
-      options[:raw_email] = @raw_email
-
-      options[:created_at] ||= @mail.date
-      options[:created_at] = DateTime.now if options[:created_at] > DateTime.now
-
+    def add_elided_to_raw!(options)
```

Comment:
This method is weird. It's returning `options` which looks like is never used and also updates "in-place" the `options` argument?

Reply:
You are right, I was torn between the two and finally went with the in-place changes and left the return code too.